engine/flink/kafka-util/src/main/scala/pl/touk/nussknacker/engine/kafka/source/KafkaSourceFactory.scala

```diff
@@ -26,6 +26,8 @@ import scala.reflect.ClassTag
   *   - KafkaSourceFactory - `topic` parameter has to be passed on frontend
   *   - SingleTopicKafkaSourceFactory - topic is defined on level of configuration
   *
+  * Used to produce simple sources with single input variable.
```

Comment:
Do we add depreaction for this class? Why someone wil use it if he/she has better approach?

Reply:
Fixed, skipped phasing out and replaced `KafkaSourceFactory`